src/test/java/ua/cv/tim/parser/VillageParserTest.java

```diff
@@ -32,33 +25,33 @@
     @Autowired
     VillageService villageService;
 
-    @Test
-    public void testPars() throws Exception {
-        HtmlUnitDriver driver = new HtmlUnitDriver();
-        driver.setJavascriptEnabled(true);
-
-        driver.get("http://ts1.travian.ru/login.php");
-        driver.findElements(By.name("name")).clear();
-        driver.findElements(By.name("name")).get(0).sendKeys("Star");
-        driver.findElements(By.name("password")).clear();
-        driver.findElements(By.name("password")).get(0).sendKeys("321654aaa");
-        driver.findElement(By.cssSelector("div.button-content")).click();
-
-        Village village = villageParser.pars(driver);
-        System.out.println(village.getName());
-        System.out.println(village.getWall());
-        System.out.println(village.getPopulation());
-        System.out.println(village.getxCoord());
-        System.out.println(village.getyCoord());
-        System.out.println(village.getIsCapital());
-        driver.quit();
-    }
-
-    @Test
-    public void testSync() throws Exception {
-        Village village = villageService.getById("1");
-        village.setName("1s222i");
-        villageParser.sync(village, "1");
-    }
+//    @Test
+//    public void testPars() throws Exception {
+//        HtmlUnitDriver driver = new HtmlUnitDriver();
+//        driver.setJavascriptEnabled(true);
+//
+//        driver.get("http://ts1.travian.ru/login.php");
+//        driver.findElements(By.name("name")).clear();
```

Comment:
This tests not passed, so Sergiy said comment them. Maybe better delete them. I will do it.